charts/infra-apps/values.yaml

```diff
@@ -53,7 +53,7 @@ ingressNginx:
   # ingressNginx.chart -- Chart
   chart: "ingress-nginx"
   # ingressNginx.targetRevision -- [ingress-nginx Helm chart](https://github.com/kubernetes/ingress-nginx/tree/master/charts/ingress-nginx) version
-  targetRevision: '3.7.*'
+  targetRevision: '3.17.*'
```

Comment:
Depracation of helm2 support + update to v0.42.0 ingress-nginx 

* https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/CHANGELOG.md#3120
* https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/CHANGELOG.md#3170